src/test/java/com/exasol/projectkeeper/validators/ProjectKeeperPluginDeclaration.java

```diff
@@ -0,0 +1,74 @@
+package com.exasol.projectkeeper.validators;
+
+import java.util.Arrays;
+import java.util.List;
+
+import com.exasol.projectkeeper.ProjectKeeperModule;
+
+public class ProjectKeeperPluginDeclaration {
```

Comment:
Can you please add a comment about what do you use this class for? Jus to help understand it better